wcfsetup/install/files/lib/data/user/UserProfile.class.php

```diff
@@ -438,23 +439,28 @@ public static function getUserProfilesByUsername(array $usernames) {
 	 */
 	public function isAccessible($name) {
 		/** @noinspection PhpVariableVariableInspection */
+		$data = ['result' => true, 'name' => $name];
+		
 		switch ($this->$name) {
 			case self::ACCESS_EVERYONE:
-				return true;
+				$data['result'] = true;
 			break;
 			
 			case self::ACCESS_REGISTERED:
-				return (WCF::getUser()->userID ? true : false);
+				$data['result'] = (WCF::getUser()->userID ? true : false);
 			break;
 			
 			case self::ACCESS_FOLLOWING:
-				return ($this->isFollowing(WCF::getUser()->userID) ? true : false);
+				$data['result'] = ($this->isFollowing(WCF::getUser()->userID) ? true : false);
 			break;
 			
 			case self::ACCESS_NOBODY:
-				return false;
+				$data['result'] = false;
 			break;
 		}
+		
+		EventHandler::getInstance()->fireAction($this, 'isAccessible', $data);
+		return $data['result'];
```

Comment:
Please add a blank line between the `EventHandler` call and the return statement, this is true for the other, similar occurrences below.